common/elasticsearch/src/main/scala/uk/ac/wellcome/elasticsearch/WorksIndexConfig.scala

```diff
@@ -272,4 +280,12 @@ object IndexedWorkIndexConfig extends WorksIndexConfig {
         keywordField("info")
       )
     )
+
+  /**
+    * Denormalised relations make index sizes explode from about 5gb without relations
```

Comment:
It is worth mentioning that ~5GB on the denormalised index becomes ~25GB in the catalogue index due the mapping of all the extra fields.